### PR TITLE
fix(design): small fixes to the UI design

### LINKED ISSUE #1367: Minor UI Fixes
#  Search Suggestions

The appearance of the search suggestions needs to be cleaned up.

image

# Previous/Next

Previous/Next should be replaced with `<` and `>`.

image

# Mouse Over on Buttons

Cannot read the text on buttons when the mouse is over it.

image

# Dataset
The content should be closer to the titles.

image

# Project Dataset Listing

The project dataset listing should use the tiles

image

# Misc other fixes

- Make sure we are using `success` and not `rk-green`
- Button text alignment
- Dataset "add to button project" should be green
- Check that everything works on smaller screen sizes

### PR DESCRIPTION
**Pull request by vfried**
change to text size in help page

image

fix to menu in environments
image

image

Pagination now looks like:
image

Warning sign style:
before
image

after
image

Dataset details table has less spacing 
before
image

after
image

Action buttons in datasets are now green
image

image

/deploy renku=ui-design-2021

Closes #1367

### CODE REVIEW

File: client/src/utils/UIComponents.js [hunk 1]
```diff
@@ -198,12 +198,10 @@ class Pagination extends Component {
       innerClass={className}
 
       // Some defaults for the styling
-      prevPageText={"Previous"}
-      nextPageText={"Next"}
       itemClass={"page-item"}
       linkClass={"page-link"}
       activeClass={"page-item active"}
-      hideFirstLastPages={false}
+      hideFirstLastPages={true}
```
Comment (ciyer) [inline client/src/utils/UIComponents.js:204]: Sorry, I think there was a misunderstanding in the design. The switch to "<" and ">" for previous/next is great, but the first and last page buttons should stay.

```suggestion
      hideFirstLastPages={false}
```

**Review by ciyer — CHANGES_REQUESTED** [review]
These are very nice fixes. One small correction request on a fix:

- Please bring back the pagination options to go to the beginning/end of the list

I think two of the improvements identified in the issue were missed:

## Dataset

The content should be closer to the titles. This has been fixed in the project/dataset page, but not the global dataset page.

image

## Mouse Over on Buttons

Cannot read the text on buttons when the mouse is over it.

image

**Comment by vfried** [comment]
Fixed outlined primary button 
image

Pagination fixed --> I misunderstood what you wanted there

Datasets --> no fix needed the component is the same in both places (hard refresh should fix this)

General search
image

Project
image

**Comment by vfried** [comment]
Added loader to datasets search/list --> i had removed it accidentally 🤦‍♀️

**Review by ciyer — APPROVED** [review]
👍

**Pull request merged**